drom.toml

```diff
@@ -33,7 +33,7 @@ of the superbol-studio-oss OCaml project
 """
 
 [project]
-skip = ["@test", "@ocamlformat", "@ocp-indent", "README.md", "sphinx/about.rst", "LICENSE.md", "sphinx/index.rst"]
+skip = ["@test", "@ocamlformat", "@ocp-indent", "README.md", "sphinx/about.rst", "LICENSE.md", "sphinx/index.rst", "src/lsp/pretty/main.ml", "src/lsp/cobol_typeck/main.ml", "src/lsp/cobol_ptree/main.ml", "src/lsp/cobol_lsp/main.ml", "src/lsp/cobol_lsp/main.ml", "src/lsp/cobol_data/main.ml", "src/lsp/cobol_indent/main.ml", "src/testing/superbol_testutils/main.ml"]
```

Comment:
I myself had many more entries here not so long ago… wondering what happened.

I'll try to find them.

Reply:
Actually that was in an earlier version/different branch of this project.

Reply:
Maybe we can import the `skip` list from that earlier version then?

Reply:
Could you run `opam exec -- drom project` to see if it still generates some files we don't want? If no, then that PR  can probably be merged as is.